src/PocketShareButton.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+import assert from 'assert';
+
+import createShareButton from './utils/createShareButton';
+import objectToGetParams from './utils/objectToGetParams';
+
+function pocketLink(url, { title }) {
+  assert(url, 'pocket.url');
+
+  return 'https://getpocket.com/save' + objectToGetParams({
+    url,
+    text: title,
```

Comment:
The same question over here: should the query parameter be named `title`? The links mentioned in the previous comment provide examples for Pocket. There it seems that `title` is used.

Reply:
Good catch! After reviewing the [documentation for their API](https://getpocket.com/developer/docs/v3/add) it looks like the title parameter is ignored if Pocket can get a title parameter from the shared page, so it looked like it was working for me when I tested. I'll make this change, and also I notice that Pocket supports a `tags` property and also a `tweet_id` so I'll update to add those.